src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -138,6 +138,25 @@
   ],
 
   properties: [
+    {
+      class: 'String',
+      name: 'referenceSummary',
+      section: 'requestDetails',
+      transient: true,
+      tableWidth: 200,
+      tableCellFormatter: function(_,obj) {
```

Comment:
the detailed view also has to be set, not only the table cell

Reply:
we can do this in a followup

Reply:
Done.